ui/api/topics/actions.ts

```diff
@@ -38,6 +40,7 @@ export async function getTopics(
         "name,status,visibility,numPartitions,totalLeaderLogBytes,consumerGroups",
       "filter[id]": params.id ? `eq,${params.id}` : undefined,
       "filter[name]": params.name ? `like,*${params.name}*` : undefined,
+      "filter[status]": params.status,
```

Comment:
In this case where there is no "operator" in the filter like `eq` or `in`, the API defaults to `eq`. If you ever want to search for multiple status values, the `in` could be used.

Reply:
Ah it wasn't clear from the openapi doc, I thought it supported a single value only

Reply:
Yeah, I missed docs for many of the `filter` parameters. They should all follow that pattern. I'll see if there's a nice way to document that without too much repetition.

Reply:
Done, updated screens in the PR description to match the new option